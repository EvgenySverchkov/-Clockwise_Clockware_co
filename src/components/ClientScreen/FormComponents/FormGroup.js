import React from "react";

const FormGroup = ({children, isRow=true}) => (
    <div className={`form-group ${isRow ? 'row':''} text-center text-sm-left col-12 col-md-10 col-lg-8`}>
        {children}
    </div>
);

export default FormGroup;