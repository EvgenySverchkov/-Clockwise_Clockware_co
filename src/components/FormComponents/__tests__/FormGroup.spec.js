import React from 'react';
import {create} from "react-test-renderer";

import FormGroup from "../FormGroup";

it("<FormGroup /> test which is used in forms", ()=>{
    const component = create(<FormGroup>
        <div>Some element</div>
    </FormGroup>)
    expect(component.toJSON()).toMatchSnapshot();
});