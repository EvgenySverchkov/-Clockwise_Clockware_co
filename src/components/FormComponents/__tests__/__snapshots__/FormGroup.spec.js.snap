// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormGroup /> test which is used in forms 1`] = `
<div
  className="form-group row text-center text-sm-left col-12 col-md-10 col-lg-8"
>
  <div>
    Some element
  </div>
</div>
`;
